src/data/showcases.js

```diff
@@ -1345,7 +1345,7 @@ export const Showcases = [
     website: "https://github.com/txpipe/asteria",
     source: "https://github.com/txpipe/asteria",
     tags: ["game", "opensource"],
-  },
+  }
```

Comment:
```suggestion
  },
```
We _definitely_ need to leave the removed comma in here to preserve the symmetry of array items.